### PR TITLE
Make blue really blue

### PR DESCRIPTION
**Pull request by wxiaoguang**
The `var(--color-primary)` shouldn't be used for color blue, otherwise the blue becomes green in some themes.

### CODE REVIEW

File: web_src/less/_base.less [hunk 1]
```diff
@@ -893,13 +893,13 @@ a.ui.card:hover,
...
     &.blue {
-      color: var(--color-primary) !important;
+      color: var(--color-blue) !important;
 
       a {
         color: inherit !important;
 
         &:hover {
-          color: var(--color-primary-dark-1) !important;
+          color: #3876b3 !important;
```
Comment (6543) [inline web_src/less/_base.less:902]: we should move this into css var ...

Comment (wxiaoguang) [reply web_src/less/_base.less:902]: Not this time. It just follows the `red` above.


image

**Review by 6543 — APPROVED** [review]

**Comment by silverwind** [comment]
There's a number of buttons in templates that use `blue` instead of `primary`, so these should regress to blue after this change. I think these classes need to be renamed here as well.

**Comment by zeripath** [comment]
> There's a number of buttons in templates that use `blue` instead of `primary`, so these should regress to blue in arc-green after this change. I think these classes need to be renamed here as well.

There are 95 results in 62 *.tmpl files...

**Comment by wxiaoguang** [comment]
> There's a number of buttons in templates that use `blue` instead of `primary`, so these should regress to blue in arc-green after this change. I think these classes need to be renamed here as well.

Will you propose a PR to replace this one? Then close this one.

**Comment by zeripath** [comment]
I have a feeling that a simple search and replace will work fine

**Comment by wxiaoguang** [comment]
I'm open, this PR can be edited directly, or accept new PRs for it.

**Comment by silverwind** [comment]
I'll have a look.

**Comment by silverwind** [comment]
https://github.com/go-gitea/gitea/pull/19763

**Pull request closed without merging**